staging/src/kubevirt.io/client-go/api/v1/openapi_generated.go

```diff
@@ -15968,7 +15968,7 @@ func schema_kubevirtio_client_go_api_v1_VirtualMachine(ref common.ReferenceCallb
 						},
 					},
 				},
-				Required: []string{"spec"},
+				Required: []string{"spec", "status"},
```

Comment:
Does this mean `status` is a required field in a VM resource?

Reply:
@omeryahud Yes and it is indeed not logical. I will try to overcome that. My intention was to give initial values to status fields w/o using mutating webhooks, also I can't do it at the CRD level (tried to play with expressions in JSONPath)

Reply:
@omeryahud Fixed. it is not required now. And status didn't get harm